client/src/main/java/sigblockchain/projectred/client/Account.java

```diff
@@ -0,0 +1,64 @@
+package sigblockchain.projectred.client;
+
+import java.math.BigInteger;
+
+public class Account {
+	private String WalletAddress;
+	private BigInteger Balance;
+	private BigInteger StateNonce;
+
+	public Account(String walletAddress, BigInteger balance, BigInteger stateNonce) {
+		validateWalletAddress(walletAddress);
```

Comment:
For now you don't need to validate the wallet address. But you do need to validate balance and state nonce, as neither value can be negative.

Reply:
Would you rather I comment out the wallet address validation as a placeholder for the future, or remove it altogether?